BUILDING_ON_LINUX.md

```diff
@@ -4,27 +4,28 @@ Note on Qt version compatibility: If you are installing Qt from a package manage
 
 ## Ubuntu 18.04
 
-_most likely works the same for other Debian-like distros_
+_Most likely works the same for other Debian-like distros_
 
-1. Install dependencies `sudo apt install qttools5-dev qtmultimedia5-dev libqt5svg5-dev libboost-dev libssl-dev libboost-system-dev libboost-filesystem-dev cmake g++`
+1. Install all of the dependencies using `sudo apt install qttools5-dev qtmultimedia5-dev libqt5svg5-dev libboost-dev libssl-dev libboost-system-dev libboost-filesystem-dev cmake g++`
 
-### Through Qt Creator
+### Compiling through the Qt Creator
```

Comment:
Don't see why "the" is needed here.

Reply:
*the* used with a non-specific noun is incorrect